pkg/authorization/api/types.go

```diff
@@ -20,18 +20,53 @@ const (
 	VerbAll     = "*"
 )
 
+const (
+	ResourceGroupPrefix         = "resourcegroup"
```

Comment:
> On a related note, we should keep a list of all the things someone adding a new resource type needs to do, and put these things on that list:

I agree, but policy is just a small part of that list that includes: origin latest, client, describers, printers, storage, etc.  Where should it live?